substra/sdk/backends/local/backend.py

```diff
@@ -602,7 +602,10 @@ def _output_from_spec(outputs: Dict[str, schemas.ComputeTaskOutputSpec]) -> Dict
 
 
 def _schemas_list_to_models_list(inputs: Any, model: Any) -> Any:
-    return [model.model_validate(input_schema.model_dump()) for input_schema in inputs]
+    if inputs:
```

Comment:
can `inputs` be None? 
If it's always an iterable, you don't need the conditional

Reply:
Yes, for instance in `TaskSpec`, inputs is given as "optional". In practice, it does not looks lite to be none sometimes but I added the condition to match the typing

Reply:
minor but I feel the flow is slightly more readable with 
```
if not inputs: 
    return []
return [model.model_validate(input_schema.model_dump()) for input_schema in inputs]
```

(reduce nested statements)